io.github.obsidian-level-maker.yml

```diff
@@ -0,0 +1,44 @@
+app-id: io.github.obsidian-level-maker
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: obsidian.sh
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=x11
```

Comment:
it's either or. of the too.

You mention fltk, but it's not built either.

Reply:
A vendored version of FLTK from our repository is both included and built.

Reply:
then `socket=x11` should be removed if it supports wayland *out of the box*.

Reply:
Done